src/test/java/co/org/osso/api/ApliApplicationCalculatorTests.java

```diff
@@ -0,0 +1,60 @@
+package co.org.osso.api;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ApliApplicationCalculatorTests {
+
+    CalculatorService CalculatorServiceTarget = new CalculatorService();
```

Comment:
just call this `target`. Remember, variables (that includes fields!) should have be named using `camelCase`